app.css

```diff
@@ -14,66 +14,69 @@
   justify-content: flex-start;
 }
 
-.nav-icon {
+.toggleUl {
+  display: flex;
+  flex-direction: column;
+  width: 100vw;
+  height: 100vh;
   position: fixed;
-  align-self: flex-end;
-  padding-bottom: 10em;
-  margin: 20px;
-  z-index: 9999;
+  justify-content: center;
+  align-items: center;
+  background-color: #fff;
+  opacity: 0.95;
+  background-image: url(images/toggle1.png), url(images/toggle2.png);
+  background-repeat: no-repeat;
+  background-size: 100%;
+  background-position: top, bottom;
+  list-style: none;
+}
+
+.navbar ul {
+  display: none;
 }
 
-.nav-icon div {
-  width: 25px;
-  height: 4px;
-  background-color: #000;
-  margin: 4px;
-  transition: all 0.3s ease;
+.menu-list {
+  list-style-type: none;
 }
```

Comment:
There is an issue with the functionality of the hamburger button. When you close it, the desktop menu popups as you will see in the following image. Please be sure to fix this so you meet the project requirements 🙌

![image](https://user-images.githubusercontent.com/15807343/151901686-e1ac863b-3a01-4377-b64c-f58643038198.png)

Reply:
Bro can you help to fix it? I have tried everything but nothing has worked

Reply:
Just got your message 😥Good to know you found a solution!